tests/foreman/cli/test_hammer.py

```diff
@@ -0,0 +1,78 @@
+"""Tests related to hammer command and its options and subcommands."""
+import json
+
+from robottelo.cli import hammer
+from robottelo.common import ssh
+from robottelo.common.helpers import read_data_file
+from robottelo.test import CLITestCase
+
+HAMMER_COMMANDS = json.loads(read_data_file('hammer_commands.json'))
```

Comment:
Is it likely that the contents of this file will ever change at runtime? If so, it may be better to define a function that reads and returns the file each time it is called, rather than reading the file once when the module is imported.